Add paginator

Masdestructor opened the pull request:
Ajout d'un système de page dans les tableaux pour pas avoir toutes les données sur une page. 
![image](https://github.com/user-attachments/assets/406ace19-97ca-4093-bc64-dd203c411818)

Masdestructor committed:
add Paginator

Ajout de paginator pour faire des pages dans le tableau avec les donnée pour ne pas tout avoir sur une même page.

Masdestructor committed:
Revert "add Paginator"

This reverts commit 67d1052bc14b082532c3d08e84f5548bd8fcbdcb.

GeoffreyRe commented:
bonjour vincent, tu as créé un commit avec tes changements, puis un revert de ce comit. De ce fait, il n'y a rien à merger, est-ce normal ?

Masdestructor committed:
add Paginator

Ajout de paginator pour faire des pages dans le tableau avec les donnée pour ne pas tout avoir sur une même page.

Masdestructor commented:
Bonjour Geoffrey, 

Je viens de voir que j'ai fait une mauvaise manipulation du coup j'ai mis à jour le code pour le système de page.
